Cargo.toml

```diff
@@ -1,7 +1,33 @@
+# Copyright 2018 Bitwise IO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
 [package]
 name = "sawtooth-pbft"
 version = "0.1.0"
 authors = ["Bitwise IO, Inc"]
 
 [dependencies]
-sawtooth_sdk = { git = "https://github.com/hyperledger/sawtooth-core.git" }
+sawtooth_sdk = { git = "https://github.com/bridger-herman/sawtooth-core.git", branch = "pbft-wip" }
```

Comment:
What SDK changes require this? Can we get them into the official repo?

Reply:
`pbft-wip` is forked off of @aludvik's `raft-prototype-demo` branch, which has an older version of the Consensus API in it. The only change from that branch was to add a PBFT liveness test. Once the Consensus API is in a more final form, PBFT will use that.